mods/ts/rules/defaults.yaml

```diff
@@ -41,18 +51,33 @@
 		TerrainTypes: Clear, Road, DirtRoad, Rough
 		Adjacent: 4
 	FrozenUnderFog:
-	GivesBuildableArea:
-	Capturable:
 	SoundOnDamageTransition:
 		DamagedSounds: expnew01.aud
 		DestroyedSounds: crmble2.aud
 	RenderBuilding:
 	WithBuildingExplosion:
+	RepairableBuilding:
```

Comment:
This lets civilian buildings be repaired via the mouse. I thought they were only repairable via engineers in TS as well.